plugins/discovery-ec2/src/main/java/org/elasticsearch/discovery/ec2/AwsEc2UnicastHostsProvider.java

```diff
@@ -64,6 +66,9 @@
 
     private final HostType hostType;
 
+    private final DiscoNodesCache discoNodes;
+    private final static String NODE_CACHE_TIME_SETTING = "discovery.ec2.node_cache_time";
```

Comment:
Can you document this new `discovery.ec2.node_cache_time` setting in https://github.com/elastic/elasticsearch/blob/master/docs/plugins/discovery-ec2.asciidoc?
There is a table with all supported settings in `EC2 Discovery` section.